pymc3/tests/test_distributions.py

```diff
@@ -2909,9 +2909,12 @@ class TestBugfixes:
         "dist_cls,kwargs", [(MvNormal, dict(mu=0)), (MvStudentT, dict(mu=0, nu=2))]
     )
     @pytest.mark.parametrize("dims", [1, 2, 4])
-    @pytest.mark.xfail(reason="Distribution not refactored yet")
     def test_issue_3051(self, dims, dist_cls, kwargs):
-        d = dist_cls.dist(**kwargs, cov=np.eye(dims), size=(dims,))
+        mu = np.repeat(kwargs["mu"], dims)
```

Comment:
Not very important but this would also work:

```python
    @pytest.mark.parametrize(
        "dist_cls,kwargs", [(MvNormal, dict()), (MvStudentT, dict(nu=2))]
    )
    @pytest.mark.parametrize("dims", [1, 2, 4])
    def test_issue_3051(self, dims, dist_cls, kwargs):
        mu = np.repeat(0, dims)
        d = dist_cls.dist(mu=mu, cov=np.eye(dims), **kwargs, size=20)
```